.github/workflows/publish.yml

```diff
@@ -30,3 +30,45 @@ jobs:
         run: |
           echo "🚀 Publishing cargo package"
           cargo make --env CARGO_API_TOKEN="${{ secrets.CARGO_API_TOKEN }}" -- publish
+
+  publish-docker-images:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Set up context
+        id: project_context
+        uses: FranzDiebold/github-env-vars-action@v2.4.0
+
```

Comment:
```suggestion
```

Used to generate the image tag for PR, but not needed as we publish only main builds